contrib/cirrus/ts.awk

```diff
@@ -0,0 +1 @@
+{print strftime("[%s]"), $0}
```

Comment:
Oh please no. Can you try %H:%M:%S ? Humans can't read time_t.

Reply:
As a human I agree.

Reply:
I agree no time_t, but there's one better:  We only really care about the time-delta.  I'm experimenting to see if I can get it presented in a readable way.  If not HMS is the fallback.